exchanges/binance/binance.go

```diff
@@ -1267,3 +1299,475 @@ func (b *Binance) FetchSpotExchangeLimits(ctx context.Context) ([]order.MinMaxLe
+	params.Set("loanCoin", loanCoin.String())
+	if loanAmount != 0 {
+		params.Set("loanAmount", strconv.FormatFloat(loanAmount, 'f', -1, 64))
+	}
+	params.Set("collateralCoin", collateralCoin.String())
+	if collateralAmount != 0 {
+		params.Set("collateralAmount", strconv.FormatFloat(collateralAmount, 'f', -1, 64))
+	}
+
+	var resp FlexibleLoanBorrow
+	return &resp, b.SendAuthHTTPRequest(ctx, exchange.RestSpotSupplementary, http.MethodPost, flexibleLoanBorrow, params, spotDefaultRate, &resp)
+}
+
+// FlexibleLoanOngoingOrders gets the flexible loan ongoing orders
+func (b *Binance) FlexibleLoanOngoingOrders(ctx context.Context, loanCoin, collateralCoin currency.Code, current, limit int64) (*FlexibleLoanOngoingOrder, error) {
+	params := url.Values{}
+	if !loanCoin.IsEmpty() {
+		params.Set("loanCoin", loanCoin.String())
+	}
+	if !collateralCoin.IsEmpty() {
+		params.Set("collateralCoin", collateralCoin.String())
+	}
+	if current != 0 {
+		params.Set("current", strconv.FormatInt(current, 10))
+	}
+	if limit != 0 {
+		params.Set("limit", strconv.FormatInt(limit, 10))
+	}
+
+	var resp FlexibleLoanOngoingOrder
+	return &resp, b.SendAuthHTTPRequest(ctx, exchange.RestSpotSupplementary, http.MethodGet, flexibleLoanOngoingOrders, params, spotDefaultRate, &resp)
+}
+
+// FlexibleLoanBorrowHistory gets the flexible loan borrow history
+func (b *Binance) FlexibleLoanBorrowHistory(ctx context.Context, loanCoin, collateralCoin currency.Code, startTime, endTime time.Time, current, limit int64) (*FlexibleLoanBorrowHistory, error) {
+	params := url.Values{}
+	if !loanCoin.IsEmpty() {
+		params.Set("loanCoin", loanCoin.String())
+	}
+	if !collateralCoin.IsEmpty() {
+		params.Set("collateralCoin", collateralCoin.String())
+	}
+	if !startTime.IsZero() {
+		params.Set("startTime", strconv.FormatInt(startTime.UnixMilli(), 10))
+	}
+	if !endTime.IsZero() {
+		params.Set("endTime", strconv.FormatInt(endTime.UnixMilli(), 10))
+	}
+	if current != 0 {
+		params.Set("current", strconv.FormatInt(current, 10))
+	}
+	if limit != 0 {
+		params.Set("limit", strconv.FormatInt(limit, 10))
+	}
+
+	var resp FlexibleLoanBorrowHistory
+	return &resp, b.SendAuthHTTPRequest(ctx, exchange.RestSpotSupplementary, http.MethodGet, flexibleLoanBorrowHistory, params, spotDefaultRate, &resp)
+}
+
+// FlexibleLoanRepay repays a flexible loan
+func (b *Binance) FlexibleLoanRepay(ctx context.Context, loanCoin, collateralCoin currency.Code, amount float64, collateralReturn, fullRepayment bool) (*FlexibleLoanRepay, error) {
+	if loanCoin.IsEmpty() {
+		return nil, errLoanCoinMustBeSet
+	}
+	if collateralCoin.IsEmpty() {
+		return nil, errCollateralCoinMustBeSet
+	}
+	if amount <= 0 {
+		return nil, errAmountMustBeSet
+	}
+
+	params := url.Values{}
+	params.Set("loanCoin", loanCoin.String())
+	params.Set("collateralCoin", collateralCoin.String())
+	params.Set("repayAmount", strconv.FormatFloat(amount, 'f', -1, 64))
+	if collateralReturn {
+		params.Set("collateralReturn", "true")
+	}
```

Comment:
> Default: TRUE. TRUE: Return extra collateral to spot account

[source](https://binance-docs.github.io/apidocs/spot/en/#repay-flexible-loan-repay-trade)

```suggestion
	if !collateralReturn {
		params.Set("collateralReturn", "false")
	}
```
You might need some other reworking to allow false to be set instead of just default value